configure.ac

```diff
@@ -131,11 +131,22 @@ if test "x$AGENTS_LIST" = x; then
 fi
 
 if test "x$AGENTS_LIST" != xall; then
+	TMP=""
 	for j in $AGENTS_LIST; do
 		if ! test -d fence/agents/$j; then
 			AC_ERROR([Agent $j does not exists])
 		fi
+
+		if test "$j" = "lib" -o "$j" = "nss_wrapper" -o "$j" = "autodetect"; then
+			continue
+		fi
+
+		AGENT=`find $srcdir/fence/agents/$j -name '*.py' -printf '%P'`
```

Comment:
So the only problem is how to find out what to build. I'm not against *py version because it will cover majority of use cases but we need a backup for the other cases. imho it should be in makefile - @oalbrigt what do you think?